tasks/WhatisThat.tex

```diff
@@ -0,0 +1,64 @@
+\section{What is That?}
+\label{test:what-is-that}
+A guest asks the robot to identify objects in the household.
+
+\noindent \textbf{Focus:} \SysI{}, \HRI{}, \GestRec{}, \CV{}, \OR{}.
+
+
+\subsection*{Main Goal}
+The robot names 5 objects, the operator is pointing at.
```

Comment:
Maybe change to "The robot names 5 objects that the guest points at."

Reply:
Guest is definitely better. Question for native speaker:
The robot names 5 objects, the guest is pointing at. 
or
The robot names 5 objects that the guest is pointing at. 
?